assembly/Int/CompactInt.ts

```diff
@@ -12,32 +12,34 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { Bytes } from "../utils/Bytes";
-import { Codec } from "../interfaces/Codec";
-import { BIT_LENGTH } from "../utils/Bytes";
+import { UnwrappableCodec } from "../interfaces/UnwrappableCodec";
+import { BIT_LENGTH, Bytes } from "../utils/Bytes";
 import { BytesBuffer } from "../utils/BytesBuffer";
 
 /** Representation for a Int8 value in the system. */
```

Comment:
`Representation for a Int8 value in the system` -> `Representation for a Compact Int value in the system`